Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9-buster
+RUN mkdir /app
+COPY androgee /app/androgee
+COPY poetry.lock /app
+COPY pyproject.toml /app
+RUN apt update && apt upgrade -y
+RUN pip3 install poetry
+WORKDIR /app
+RUN poetry install
+ENV DISCORD_PREFIX setthese
+ENV DISCORD_TOKEN 
+ENV mod_role_id
+ENV mod_role_name
```

Comment:
We can leave these ENV's out because I deploy containers with Salt and define the environment vars there. Reference: https://docs.saltproject.io/en/latest/ref/states/all/salt.states.docker_container.html#salt.states.docker_container.running

Reply:
I've changed it